app/storage/dynamodb.py

```diff
@@ -20,14 +26,15 @@ def put(self, model, overwrite=True):
             response = table.put_item(**put_kwargs)["ResponseMetadata"][
                 "HTTPStatusCode"
             ]
-            return response == 200
+            succeeded: bool = response == 200
+            return succeeded
         except ClientError as e:
             if e.response["Error"]["Code"] == "ConditionalCheckFailedException":
                 raise ItemAlreadyExistsError() from e
 
             raise  # pragma: no cover
 
-    def get(self, model_type, key_value):
+    def get(self, model_type: Type[ModelTypes], key_value: str) -> Optional[ModelTypes]:
```

Comment:
why `Type` ?

Reply:
Because we use the type to look u the model. It is not an instance here, e.g:
https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/data_models/session_store.py#L87